crates/runtime/src/lib.rs

```diff
@@ -65,6 +69,27 @@ impl Runtime {
             },
         }
     }
+
+    pub fn start_pods_watcher(&self) -> notify::Result<()> {
+        let mut current_app = self.app.clone();
```

Comment:
```suggestion
        let mut current_app = Arc::clone(&self.app);
```

I find this syntax more explicit.